com.prof18.feedflow.yml

```
@@ -0,0 +1,73 @@
+app-id: com.prof18.feedflow
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk21
+command: feedflow
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=xdg-documents:ro
+  - --filesystem=xdg-download:rw
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk21/install.sh
+
+  - name: feedflow
+    buildsystem: simple
+    build-options:
+      append-path: "/usr/lib/sdk/openjdk21/bin"
+    build-commands:
+      - echo "is_release=true" >> desktopApp/src/jvmMain/resources/props.properties
+      - echo "version=1.3.0" >> desktopApp/src/jvmMain/resources/props.properties
+      - echo "flatpak=true" >> desktopApp/src/jvmMain/resources/props.properties
+      - "sed -i s/distributionUrl.*/distributionUrl=gradle-bin.zip/ gradle/wrapper/gradle-wrapper.properties"
+      - |
+        sed -i \
+          -e 's/alias(libs\.plugins\.triplet\.play) apply false/\/\/ &/' \
+          -e 's/alias(libs\.plugins\.crashlytics) apply false/\/\/ &/' \
+          -e 's/alias(libs\.plugins\.google\.services) apply false/\/\/ &/' \
+          build.gradle.kts || true
```

```suggestion
          build.gradle.kts
```
If this fail then the build should fail.

The us of `sed` in general is strongly discouraged.